website/scaffolds/draft.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
If the `scaffolds` directory isn't used, we should remove it.  If it has utility, we should document it.